packages/desktop-client/src/components/manager/BudgetList.tsx

```diff
@@ -444,23 +443,24 @@ export function BudgetList({ showHeader = true, quickSwitchMode = false }) {
           }}
         >
           <Button
-            type="bare"
+            variant="bare"
+            aria-label="Import file"
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏no need to manually add an aria-label if the content of the button already contains what we want.
```suggestion
```